.travis.yml

```diff
@@ -24,6 +22,7 @@ jobs:
       deploy:
         provider: pypi
         skip_existing: true
+        # Ievgen
```

Comment:
This scared me because I didn't recognize the username, so adding a comment.

Reply:
sorry, I didn't want to confuse you